macosx/Controller.m

```diff
@@ -904,7 +904,8 @@ - (void) download: (NSURLDownload *) download decideDestinationWithSuggestedFile
 
 -(void) download: (NSURLDownload *) download didCreateDestination: (NSString *) path
 {
-    [(NSMutableDictionary *)[fPendingTorrentDownloads objectForKey: [[download request] URL]] setObject: path forKey: @"Path"];
+    NSMutableDictionary *dict = (NSMutableDictionary *)fPendingTorrentDownloads[[[download request] URL]];
```

Comment:
I'm leaning towards adjusting uncrustify.cfg to apply to objc code as well, writing docs isn't my strong suit. For now just mimick the existing code style, shouldn't be too hard (it's mostly uniform).